Phonebook.Source.MockPeopleBackend/mocks/synthetic/branches.json

```diff
@@ -1 +1 @@
449 Katarina Bridge, Raum 4B","Floor":"1","FloorPlan":"sample","Id":"6A","Number":"2B","Phone":"","Place":"Kianafort"},{"Building":"4315 Isobel Turnpike","BuildingId":76,"Description":"Kianafort, 06760 Price Throughway, Raum 2A","Floor":"1","FloorPlan":"sample","Id":"3B","Number":"5B","Phone":"","Place":"Kianafort"},{"Building":"97791 Nikolaus Gateway","BuildingId":11,"Description":"Kianafort, 96174 Towne Trail, Raum 2A","Floor":"0","FloorPlan":"sample","Id":"5B","Number":"2B","Phone":"","Place":"Kianafort"},{"Building":"06760 Price Throughway","BuildingId":21,"Description":"Kianafort, 96174 Towne Trail, Raum 2B","Floor":"3","FloorPlan":"sample","Id":"3A","Number":"6B","Phone":"","Place":"Kianafort"},{"Building":"6509 Crona Canyon","BuildingId":17,"Description":"Kianafort, 06760 Price Throughway, Raum 2A","Floor":"4","FloorPlan":"sample","Id":"6A","Number":"5B","Phone":"","Place":"Kianafort"},{"Building":"6768 Harmon Dale","BuildingId":41,"Description":"Kianafort, 4315 Isobel Turnpike, Raum 4A","Floor":"2","FloorPlan":"sample","Id":"2A","Number":"2A","Phone":"","Place":"Kianafort"},{"Building":"35039 Mante Flats","BuildingId":68,"Description":"Kianafort, 449 Katarina Bridge, Raum 6B","Floor":"5","FloorPlan":"sample","Id":"4A","Number":"5B","Phone":"","Place":"Kianafort"}]},{"ContactPerson":"","LinkRoutingWebsite":"http://maps.google.de/maps?f=q&hl=de&q=Aldaview","ReceptionFax":"1-555-761-9441","Description":"Standort Aldaview","ReceptionPhone":"(335) 178-4854","LinkPicture":"/images/sample_1280.jpg","LinkRoutingInfo":"/plans/description.pdf","City":{"Name":"Aldaview","Building":"95740 Deanna Cape","ZipCode":"79197-2639"},"RoomCollection":[{"Building":"200 Easter Crest","BuildingId":64,"Description":"Aldaview, 4510 Casper Ferry, Raum 1A","Floor":"2","FloorPlan":"sample","Id":"1A","Number":"3A","Phone":"","Place":"Aldaview"},{"Building":"2099 Grayce Haven","BuildingId":31,"Description":"Aldaview, 96174 Towne Trail, Raum 3B","Floor":"4","FloorPlan":"sample","Id":"2A","Number":"6A","Phone":"","Place":"Aldaview"},{"Building":"56106 Garett Cape","BuildingId":29,"Description":"Aldaview, 6509 Crona Canyon, Raum 4B","Floor":"1","FloorPlan":"sample","Id":"5B","Number":"5B","Phone":"","Place":"Aldaview"},{"Building":"9551 Jacobs Stravenue","BuildingId":99,"Description":"Aldaview, 8124 Torp Ways, Raum 1B","Floor":"2","FloorPlan":"sample","Id":"6B","Number":"3B","Phone":"","Place":"Aldaview"},{"Building":"449 Katarina Bridge","BuildingId":68,"Description":"Aldaview, 8733 Alfonso Port, Raum 3B","Floor":"4","FloorPlan":"sample","Id":"4A","Number":"4A","Phone":"","Place":"Aldaview"},{"Building":"96174 Towne Trail","BuildingId":84,"Description":"Aldaview, 8101 Molly Point, Raum 2A","Floor":"0","FloorPlan":"sample","Id":"4B","Number":"4A","Phone":"","Place":"Aldaview"},{"Building":"0423 Leila Heights","BuildingId":51,"Description":"Aldaview, 6509 Crona Canyon, Raum 6A","Floor":"4","FloorPlan":"sample","Id":"1A","Number":"6A","Phone":"","Place":"Aldaview"},{"Buildi
```

Comment:
please revert the changes in this file. They are not relevant for the kubernetes deployment